sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_util/configurations.py

```diff
@@ -93,6 +95,11 @@ def _default_disabled_instrumentations(configurations):
     configurations[DISABLED_INSTRUMENTATIONS_ARG] = disabled_instrumentation
 
 
+def _default_logger_name(configurations):
+    logger_name = environ.get(LOGGER_NAME_ARG, "")
+    configurations[LOGGER_NAME_ARG] = logger_name
```

Comment:
I think something got mixed up here. It's check for a "logger_name" env var instead of whether the kwargs already have it.

Reply:
Oh oops, yeah we aren't supporting env var so ill remove this.